frontend/src/scenes/cohorts/Cohort.tsx

```diff
@@ -46,6 +47,7 @@ export function Cohort({ id }: { id?: CohortType['id'] } = {}): JSX.Element {
     const { deleteCohort, setCohort, onCriteriaChange } = useActions(logic)
     const { cohort, cohortLoading } = useValues(logic)
     const { hasAvailableFeature } = useValues(userLogic)
+    const { featureFlags } = useValues(featureFlagLogic)
```

Comment:
It would be nice if `featureFlagLogic` exported its own hook,
`useFeatureFlags` that was implemented as `return useValues(featureFlagLogic);`

This allows easier refactoring in the future and hides unnecessary implementation details, but much of the system already seems to not do this, so 🤷‍♂️

Reply:
> Heya Alex, I'm not sure about the migration path here, but wondering why we need conversion utils on the frontend? Since:
> 
> 1. If you're on the "new" UI, only look at `cohort.properties`, the backend would have converted groups to adhere to new format.
> 2. If you're on "old" UI, only look at `cohort.groups`.

You're absolutely correct, there shouldn't really be a need for these conversion functions since the backend provides both. Just to confirm, the backend will handle both read and write to both `groups` and `properties` keys right?

@ELanning thanks for the thorough review! A lot of the code here is going to go stale but I'll comb through your suggestions and act on the ones that still apply.

Reply:
Good point, something that might be more concise (and what we're trying to move most of our system to) is to expose featureFlag checks as a selector in the logic data layer. This would let us do something like 

```
const { cohort, cohortLoading, isNewFeatureEnabled, ... } = useValues(logic)
```